.ci/cscs.yml

```diff
@@ -11,5 +11,35 @@ build image:
   timeout: 2h
   variables:
     DOCKERFILE: .ci/build.Dockerfile
-    PERSIST_IMAGE_NAME: $CSCS_REGISTRY_PATH/sirius/sirius-ci:$CI_COMMIT_SHA
-    DOCKER_BUILD_ARGS: '["BASE_IMAGEIMAGE}", "SPECDEV=$SPEC"]'
+    PERSIST_IMAGE_NAME: $CSCS_REGISTRY_PATH/q-e-sirius/q-e-sirius-ci:$CI_COMMIT_SHA
+    #CSCS_REBUILD_POLICY: if-not-exists
+    CSCS_REBUILD_POLICY: always
+
+.run_tests:
+  extends: .container-runner-daint-gpu
+  needs: ["build image"]
+  stage: test
+  image: $CSCS_REGISTRY_PATH/q-e-sirius/q-e-sirius-ci:$CI_COMMIT_SHA
+  variables:
+    CRAY_CUDA_MPS: 1
+    GIT_STRATEGY: none
+    MPICH_MAX_THREAD_SAFETY: multiple
+    CSCS_REGISTRY_LOGIN: 'YES'
+    PULL_IMAGE: 'YES'
+    SLURM_HINT: nomultithread
+    SLURM_UNBUFFEREDIO: ''
```

Comment:
not super useful, besides that you get thousands of almost empty lines while the image is being pulled. unbuffered io is just as good.

Reply:
I removed SLURM_UNBUFFEREDIO completely